webpack/assets/javascripts/react_app/components/ForemanModal/ForemanModalReducer.test.js

```diff
@@ -5,11 +5,13 @@ import {
   SET_MODAL_OPEN,
   SET_MODAL_CLOSED,
   ADD_MODAL,
+  SET_MODAL_START_SUBMITTING,
+  SET_MODAL_STOP_SUBMITTING,
 } from './ForemanModalConstants';
 
 const modalsState = Immutable({
-  myModal: { open: true },
-  yourModal: { open: false },
+  myModal: { open: true, submitting: true },
```

Comment:
This is exactly the way I envisioned adding functionality to ForemanModals. :+1: :+1: